src/main/java/org/opentripplanner/apis/gtfs/datafetchers/VehicleRentalStationImpl.java

```diff
@@ -96,6 +98,40 @@ public DataFetcher<Integer> vehiclesAvailable() {
     return environment -> getSource(environment).getVehiclesAvailable();
   }
 
+  @Override
+  public DataFetcher<Iterable<VehicleTypeCount>> vehicleTypesAvailable() {
+    return environment -> {
+      VehicleRentalPlace rentalPlace = getSource(environment);
+      if (rentalPlace instanceof VehicleRentalStation) {
+        VehicleRentalStation station = (VehicleRentalStation) rentalPlace;
+        return station.vehicleTypesAvailable
+          .entrySet()
+          .stream()
+          .map(vtc -> new VehicleTypeCount(vtc.getKey(), vtc.getValue()))
+          .toList();
```

Comment:
Can you move this code into the `VehicleRentalStation` so that it can be used by other APIs, too?

Also make it reusable so it can be used both for types and spaces.